### PR TITLE
Add directus IDs for basics course and create release action

### PR DESCRIPTION
**Pull request by frankharkins**
This uses the script from https://github.com/frankharkins/learning-content-tools/ to automatically upload notebooks to directus.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          python-version: 3.11
+
+      - name: Install release script
+        run: |
+          pip install git+https://github.com/frankharkins/learning-content-tools.git#subdirectory=iql-lesson-sync
+
+      - name: Publish to database
+        env:
+          DIRECTUS_DATABASE: ${{ inputs.database }}
+          DIRECTUS_TOKEN: ${{ secrets.DIRECTUS_TOKEN_${{ inputs.database }}
```
Comment (abdonrd) [inline .github/workflows/release.yml:42]:
```suggestion
          LEARNING_API_TOKEN: ${{ secrets.DIRECTUS_TOKEN_${{ inputs.database }}
```

Should be like this?

**Review by abdonrd — CHANGES_REQUESTED** [review]
`database.conf.yaml` not needed?

File: .github/workflows/release.yml (hunk 1 above)
Comment (abdonrd) [reply .github/workflows/release.yml:42]: Also replace the `secrets.DIRECTUS_TOKEN_` with `secrets.LEARNING_API_`?

Comment (frankharkins) [reply .github/workflows/release.yml]: Whoops, yes – thanks!

**Comment by frankharkins** [comment]
On hold until https://github.com/frankharkins/learning-content-tools/issues/3 is resolved.

**Comment by frankharkins** [comment]
Merging to test the action

**Pull request merged**